src/pages/Authentication/Login.jsx

```diff
@@ -0,0 +1,85 @@
+import axios from 'axios'
+import {useState} from 'react'
+import { Link, useNavigate } from 'react-router-dom'
+import "./Login.css"
+
+export default function Login() {
+  const navigate = useNavigate()
+  const [loginData, setLoginData] = useState({email: "", password: ""})
+  const [loginError, setLoginError] = useState("")
+  const [showPwd, setShowPwd] = useState(false)
+
+  const testUser = {
+    email: "theMarauders@gmail.com",
+    password: "FortunaMajor"
+  }
+
+  function setUserData(e){
+    const {name, value} = e.target
+    setLoginData(prev => ({...prev, [name]:value}))
+  }
+
+  const handleLogin = (e) => {
+    e.preventDefault()
+    sendLoginData(loginData)
+  }
+
+  const sendLoginData = async (loginData) =>{
+    try {
+      const response = await axios.post("/api/auth/login", loginData)
+      console.log(response)
+      if (response.status === 200){ 
+      const {data} = response;
```

Comment:
You can destructure this at line number 29.

`const {data} = await some api`

If you are not using the `console.log()` remove it.

Reply:
Thanks, I'll try that!